doc/getting-started/getting-started.rst

```diff
@@ -126,13 +126,20 @@ To set up the ACRN build environment on the development computer:
            e2fslibs-dev \
            pkg-config \
            libnuma-dev \
-           libcjson-dev \
            liblz4-tool \
            flex \
            bison \
            xsltproc \
            clang-format \
            bc
+         
+   .. note:: For ubuntu 18.04, if you get an error when installing ``libcjson-dev``, you need to follow these steps:
```

Comment:
```suggestion
   .. note:: For Ubuntu 18.04, if you get an error when installing ``libcjson-dev``, you need to follow these steps:
```

Reply:
Unless there are cases where Ubuntu 18.04 wouldn't need this, I would be more definite in the note and explicitly state that if you are running 18.04 you do need to follow these steps.